tests/two_dim_velocity_estimates_test.py

```diff
@@ -46,7 +46,7 @@ def update_geometry(x_grid, y_grid, model):
 def test_rad_and_pol():
     v, w = 1, 1
     ds = make_2d_realization(v, w, np.array([5, 6, 7]), np.array([5, 6, 7]))
-    pd = td.estimate_velocities_for_pixel(1, 1, ds)
+    pd = td.estimate_velocities_for_pixel(1, 1, ds, True)
```

Comment:
I did these changes (adding the True argument here) so that I can use the tests to find out why it wan't working. The tests however do not pass. 
I think you should change all Trues here to False (so that they test the improved method as they did before)
And then add one new test for the naive method.